pages/Addons.md

```diff
@@ -72,6 +72,7 @@ SfChunkInfo | FN_FAL113 | <details><summary>expand</summary>Scan the amount of s
 VillagerUtil | Apeiros-46B | <details><summary>expand</summary>VillagerUtil adds a few tools that allow for easy transportation of villagers, curing zombified villagers, cycling trades and more.</details> | [Readme](https://github.com/Apeiros-46B/VillagerUtil/blob/main/README.md) | [GitHub](https://github.com/Apeiros-46B/VillagerUtil) | [Builds](https://thebusybiscuit.github.io/builds/Apeiros-46B/VillagerUtil/main)
 Missile Warfare | Kaiyan2006 | <details><summary>expand</summary>Missile Warfare adds a bunch of missiles to Slimefun that can be fired at others!</details> | N/A | [GitHub](https://github.com/koiboi-dev/MissileWarfare) | [Releases](https://github.com/koiboi-dev/MissileWarfare/releases)
 SlimefunAdvancements | char321 | <details><summary>expand</summary>SlimefunAdvancements adds fully configurable advancements to Slimefun. Easily create your own progression!</details> | [Readme](https://github.com/qwertyuioplkjhgfd/SlimefunAdvancements/blob/main/README.md) | [Github](https://github.com/qwertyuioplkjhgfd/SlimefunAdvancements) | [Builds](https://thebusybiscuit.github.io/builds/qwertyuioplkjhgfd/SlimefunAdvancements/main/)
+SupremeExpansion | RelativoBR | <details><summary>expand</summary>Supreme Expansion is a addon to Slimefun 4 which brings various new resources to craft powerful contains items, gear and machines</details> | [Readme](https://github.com/RelativoBR/SupremeExpansion/blob/master/README.md) | [Github](https://github.com/RelativoBR/SupremeExpansion) | [Builds](https://thebusybiscuit.github.io/builds/RelativoBR/SupremeExpansion/master/)
```

Comment:
`https://thebusybiscuit.github.io/builds/RelativoBR/SupremeExpansion/master/`

404? Perhaps this has not been added to builds yet?